.dependabot/config.yml

```diff
@@ -0,0 +1,6 @@
+version: 1
+update_configs:
+  - package_manager: "dotnet:nuget"
+    directory: "/"
+    update_schedule: "weekly"
+    target_branch: "master"
```

Comment:
I think we can remove this since it says the repo's default branch is used when this isn't specified.